specs/proposals.md

```diff
@@ -35,24 +37,40 @@ are part of later specification milestones.
 
 ## Proposing L2 Output Commitments
 
-The proposer's role is to construct and submit output roots, which are commitments made on a configurable interval,
+The proposer's role is to construct and submit output roots, which are commitments to the L2's state,
 to the `L2OutputOracle` contract running on L1. It does this by running the [L2 output proposer](../op-proposer/),
 a service which periodically queries the rollup node's
 [`optimism_outputAtBlock` rpc method](./rollup-node.md#l2-output-rpc-method) for the latest output root derived
 from the latest [finalized](rollup-node.md#finalization-guarantees) L1 block. The construction of this output root is
 described [below](#l2-output-commitment-construction).
 
-If there is no newly finalized output, the service continues querying until it receives one. It then submits this
-output, and the appropriate timestamp, to the [L2 Output Root](#l2-output-root-smart-contract) contract's
-`proposeL2Output()` function. The timestamp block number must correspond to the `startingBlockNumber` plus the next
-multiple of the `SUBMISSION_INTERVAL` value.
+### L2OutputOracle v1.0.0
 
-The proposer may also delete multiple output roots by calling the `deleteL2Outputs()` function and specifying the
+The submission of output proposals is permissioned to a single account. It is
+expected that this account continues to submit output proposals over time to
+ensure that user withdrawals do not halt.
+
+The [L2 output proposer](../op-proposer) is expected to submit output roots on a
+deterministic interval based on the configured `SUBMISSION_INTERVAL` in the
+`L2OutputOracle`. The larger the `SUBMISSION_INTERVAL`, the less often L1
```

Comment:
With this change we lost any mention of:
*  the output hash being tree structured (probably ok - it wasn't specified in any detail here anyway)
* any mention of proposals being delayed if there's no new finalised output
* the specific requirement of the timestamp block number having to correspond to the `startingBlockNumber` plus the next multiple of the `SUBMISSION_NITERVAL` value. It's somewhat implied but less clear.

The original spec wasn't really precise about how things actually worked either so I'm not against this change, just wanted to check that we're happy for it to be a little less specific about the current setup.

Reply:
The output hash being tree structured is wrong? Its just a flat hash over the values, which is why I removed that. The proposer can be configured to submit only safe values or unsafe values which is implementation specific, but that could be mentioned. We no longer pass in the timestamp, but we could specify what must be passed in